src/main/resources/messages/messages.properties

```diff
@@ -239,6 +243,10 @@ crowdin.tm.download.source-language-id=Defines source language in the language p
 crowdin.tm.download.target-language-id=Defines target language in the language pair
 crowdin.tm.download.to=Path the translation memory should be downloaded to
 
+# CROWDIN BUNDLE DOWNLOAD COMMAND
+crowdin.bundle.download.id=Bundle identifier
+crowdin.bundle.usage.plain=Provide plain, processable output
```

Comment:
please use an existing string (`crowdin.list.usage.plain`) instead of creating the same one